dnf/conf/config.py

```diff
@@ -335,6 +335,12 @@ def _configure_from_options(self, opts):
                             msg = _("Main config did not have a %s attr. before setopt")
                             logger.warning(msg, name)
 
+        if getattr(opts 'debugsolver'):
+            self._set_value("debug_solver", True, dnf.conf.PRIO_COMMANDLINE)
+
+        if getattr(opts 'obsoletes'):
```

Comment:
Why did you not add 'obsoletes' in config_args?

Reply:
It's just quick fix to not forget about it :) I even did not test that it works. I'll be back on this later.

Reply:
It even contains a syntax error!